[2024-03-21T19:33:09.416-0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_multiple_tasks3.task_F manual__2024-03-21T22:33:01.059479+00:00 [queued]>
[2024-03-21T19:33:09.420-0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_multiple_tasks3.task_F manual__2024-03-21T22:33:01.059479+00:00 [queued]>
[2024-03-21T19:33:09.420-0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-03-21T19:33:09.421-0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2024-03-21T19:33:09.421-0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-03-21T19:33:09.431-0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): task_F> on 2024-03-21 22:33:01.059479+00:00
[2024-03-21T19:33:09.432-0300] {standard_task_runner.py:55} INFO - Started process 1744 to run task
[2024-03-21T19:33:09.434-0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'executing_multiple_tasks3', 'task_F', 'manual__2024-03-21T22:33:01.059479+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/multiple_tasks3.py', '--cfg-path', '/tmp/tmpn1il4ots']
[2024-03-21T19:33:09.435-0300] {standard_task_runner.py:83} INFO - Job 40: Subtask task_F
[2024-03-21T19:33:09.464-0300] {task_command.py:389} INFO - Running <TaskInstance: executing_multiple_tasks3.task_F manual__2024-03-21T22:33:01.059479+00:00 [running]> on host DESKTOP-0DNP0LN.localdomain
[2024-03-21T19:33:09.500-0300] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=falcone
AIRFLOW_CTX_DAG_ID=executing_multiple_tasks3
AIRFLOW_CTX_TASK_ID=task_F
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T22:33:01.059479+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-21T22:33:01.059479+00:00
[2024-03-21T19:33:09.501-0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-03-21T19:33:09.501-0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '']
[2024-03-21T19:33:09.504-0300] {subprocess.py:86} INFO - Output:
[2024-03-21T19:33:09.504-0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-21T19:33:09.517-0300] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=executing_multiple_tasks3, task_id=task_F, execution_date=20240321T223301, start_date=20240321T223309, end_date=20240321T223309
[2024-03-21T19:33:09.566-0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-03-21T19:33:09.574-0300] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
